assets/src/amp-validation/copy-to-clipboard-buttons.js

```diff
@@ -3,28 +3,53 @@
  */
 import Clipboard from 'clipboard';
 
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
 /**
  * Internal dependencies
  */
 import { getURLValidationTableRows } from './get-url-validation-table-rows';
 
+/**
+ * Success handler, called when data is copied to the clipboard.
+ *
+ * @param {Object} event
+ * @param {HTMLElement} event.trigger The element triggering the event.
+ */
+function onSuccess( { trigger } ) {
+	trigger.focus();
```

Comment:
Minor bug:
Clicking on the button while it's showing the updated text permanently sets it to said text:

![deepin-screen-recorder_Select area_20201028143033](https://user-images.githubusercontent.com/16200219/97480867-e9c25200-194b-11eb-89a4-e6ec62ee511a.gif)

Reply:
Yes, looking at my code, that bug makes perfect sense 🤦 Fixing.

Reply:
Thanks for the catch, @pierlon. Fixed in 47fc7be1d